Convert-ProwlarrSupportedIndexersToMarkdownTable.ps1

```diff
@@ -1,108 +1,194 @@
+## End Variables
+Write-Host 'Variables and Inputs Imported'
+
+## Build Parameters
+### App
+$api_url = ($app_baseUrl.ToString().TrimEnd('/')) + $app_api_path + $app_api_version
+$version_url = $api_url + $app_api_endpoint_version
+$indexer_url = $api_url + $app_api_endpoint_indexer
+### Github
+$gh_repo_org = $gh_app_org + '/' + $gh_app_repo + '/'
+#$gh_url = ($gh_baseUrl.ToString().TrimEnd('/')) + $gh_api_path + $gh_api_type + $gh_repo_org + $gh_api_branches + $gh_app_branch
+## Disabled; API used for manual usage only
+## 
+### Wiki
+$wiki_infolink = ($wiki_link.ToString().TrimEnd('/')) + $wiki_app_path + '/' + $wiki_page + $wiki_bookmark
+$wiki_commiturl = ($gh_web.ToString().TrimEnd('/')) + '/' + $gh_repo_org + $gh_web_commit
+Write-Host 'Parameters Built'
+
+## Invoke Requests
+#Write-Host 'Getting Github Commit Info'
+#$github_req = Invoke-WebRequest -Uri $gh_url -ContentType 'application/json' -Method Get
+#Write-Host 'Got Github Commit Info'
+## Disabled; API used for manual usage only
+Write-Host 'Getting App Version'
+$version_req = Invoke-WebRequest -Uri $version_url -Headers $headers -ContentType 'application/json' -Method Get
+Write-Host 'Got App Version'
+Write-Host 'Getting Indexer Data'
+$indexer_req = Invoke-WebRequest -Uri $indexer_url -Headers $headers -ContentType 'application/json' -Method Get
+Write-Host 'Got Indexer Data'
+
+## Convert to Objects
+Write-Host 'Converting Responses to Objects'
+#$github_obj = $github_req.Content | ConvertFrom-Json
+## Disabled; API used for manual usage only
+$version_obj = $version_req.Content | ConvertFrom-Json
+$indexer_obj = $indexer_req.Content | ConvertFrom-Json
+$indexer_name_exp = { IF ($_.IndexerUrls) { '[' + $_.name + '](' + ($_.IndexerUrls[0]) + ')' + '{#' + $_.infoLink.Replace($wiki_infolink.ToString(), '') + '}' } Else { $_.name + '{#' + $_.infoLink.Replace($wiki_infolink.ToString(), '') + '}' } }
+
+## Determine Commit from GH
+#Write-Host 'Determining Commit'
+#$commit = $github_obj | Select-Object -ExpandProperty 'commit' | Select-Object -ExpandProperty 'sha' | Out-String
+#$commit = $commit -replace "`n |`r", ''
+## Disabled; API used for manual usage only
+Write-Host "Commit is $commit"
+
+
+## Determine Version (Build)
+Write-Host 'Determining Build'
+$build = $version_obj | Select-Object -ExpandProperty 'version' | Out-String
+$build = $build -replace "`n|`r", ''
+Write-Host "Build is $build"
+
+Write-Host 'Ingesting Indexer Data'
+## Get Indexer Data
+$indexer_tbl_obj = $indexer_obj | Sort-Object -Property 'name'
+
+## Build Table Fields
+Write-Host 'Building Indexer Tables'
+### Public Usenet (Disabled)
+#Write-Host 'Building: Usenet - Public'
+#$tbl_PubUse= $indexer_tbl_obj | Where-Object {($_.privacy -eq "public") -and ($_.protocol -eq "usenet")} | `
+#         Select-Object @{Name = 'Indexer'; Expression =$indexer_name_exp}, `
 #         @{Name = 'Language'; Expression = {$_.language}}, `
 #         @{Name = 'Description'; Expression = {$_.description}}
```

Comment:
Why disable it - just have it followed by `if ( $null -eq $tbl_PubUse ) { $tbl_PubUse = "None" }` and call that later on